### PR TITLE
Support stream output

### PR DESCRIPTION
**Pull request by xepozz**
Implements https://github.com/yiisoft/yii-dev-panel/issues/13
Related https://github.com/yiisoft/yii-debug-api/pull/117

### CODE REVIEW

**Review by vjik — COMMENTED** [review]
Tests is fail... and need tests for new code.

**Review by samdark — APPROVED** [review]
Need a CHANGELOG line + fixing tests.

File: src/SapiEmitter.php [hunk 1]
```diff
@@ -109,8 +109,17 @@ private function emitBody(ResponseInterface $response): void
...
         while (!$body->eof()) {
-            echo $body->read($this->bufferSize);
+            $output = $body->read($this->bufferSize);
+            if ($output === '') {
+                continue;
+            }
+            echo $output;
+            // flush the output buffer and send echoed messages to the browser
+            while (ob_get_level() > $level) {
+                ob_end_flush();
```
Comment (vjik) [inline src/SapiEmitter.php:121]: [Seems](https://app.codecov.io/gh/yiisoft/yii-runner-http/pull/50/blob/src/SapiEmitter.php#L121), it's not covered by tests.

**Review by vjik — CHANGES_REQUESTED** [review]
Need changelog

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**